chai-immutable.js

```diff
@@ -540,5 +540,97 @@
     assert.sizeOf = function (collection, expected) {
       new Assertion(collection).size(expected);
     };
+
+    /**
+     * ### .property(path, value)
+     *
+     * Asserts that property specified exists, and if a value specified, is equal
+     * to the given value.
+     *
+     * ```js
+     * var obj = Immutable.fromJS({
+     *   left: 1,
+     *   right: {
+     *     left: 2
+     *     right: 3
+     *   }
+     * });
+     *
+     * assert.property(obj, 'left');
+     * assert.propertyVal(obj, 'left', 1);
+     *
+     * assert.deepProperty(obj, ['right', 'right']);
+     * assert.deepPropertyVal(obj, ['right', 'right'], 3);
+     * ```
+     *
+     * @name property
+     * @param {path} either the single propery name or array used for `getIn()`
+     * @param {value} expected value found at that location
+     * @api public
+     */
+    function assertProperty(_super) {
+      return function (path, val, msg) {
```

Comment:
Note that providing a custom message is not available in any other assertion of this plugin. I would prefer we don't include it in this PR for this assertion only.
If you really need custom messages, you can open an issue to solve that globally and we'll start with this assertion.

Does that make sense?